### PR TITLE
Adding 8 vs 9 info with codeblocks

### PR DESCRIPTION
**Pull request by tpmccallum**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

Screenshot 2023-02-17 at 4 23 49 pm

- [x] Has run `npm run build-index`
- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)

Screenshot 2023-02-17 at 4 25 09 pm

- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors

Screenshot 2023-02-17 at 4 24 24 pm

### CODE REVIEW

**Comment by tpmccallum** [comment]
Just FYI; replaces https://github.com/fermyon/developer/pull/346

**Review by fibonacci1729 — APPROVED** [review]

File: content/spin/spin-oci.md [hunk 1]
```diff
@@ -58,37 +59,92 @@ $ cd spin-react-fullstack
...
+{{ tabs "spin-version" }}
+
+{{ startTab "v0.8.0"}}
 
  <!-- @selectiveCpy -->
 
  ```bash
  # make sure you've built your application with `spin build` prior
-$ spin registry run ghcr.io/USERNAME/spin-react-fullstack
+ spin up --from-registry ghcr.io/USERNAME/spin-react-fullstack:v1
```
Comment (itowlson) [inline content/spin/spin-oci.md:130]: This looks like an 0.8 code block, in which case this is _not correct_.  0.8 has not changed; it remains `spin oci run`.

File: content/spin/spin-oci.md [hunk 2]
```diff
@@ -58,37 +59,92 @@ $ cd spin-react-fullstack
...
 ```
 
+{{ blockEnd }}
+
+{{ startTab "v0.9.0"}}
+
+<!-- @selectiveCpy -->
+
+ ```bash
+ # make sure you've built your application with `spin build` prior
```
Comment (itowlson) [inline content/spin/spin-oci.md:140]: `spin build` doesn't make sense for running from a registry.  A registry, like a bindle, contains a built, runnable application.

**Review by itowlson — COMMENTED** [review]
This seems to change one of the 0.8 examples in a way that makes it incorrect for that version (though it looks like another PR had previously changed it so it was already incorrectly tracking 0.9).  Spin 0.8 remains on `spin oci run`.

Also noted a comment that I think has been copied over several generations from a `spin up` example and doesn't fit in this context.

**Pull request merged**